programmers/난이도별/level02.카펫/Jaewon0702.py

```diff
@@ -0,0 +1,12 @@
+import math
+def solution(brown, yellow):
+    row=int((2+brown/2+math.sqrt((2+brown/2)**2-4*(brown+yellow)))/2) #근의 공식
+    return[row,int(brown/2-row+2)]

```

Comment:
```suggestion
    row = int((2 + brown / 2 + math.sqrt((2 + brown / 2) ** 2 - 4 * (brown + yellow))) / 2) #근의 공식
    return [ row, int(brown / 2 - row + 2) ]
```

가독성이 너무 안 좋네요.. ㅠㅠ 이렇게 띄어쓰기만 해줘도 어느정도 보기 수월해진답니다

Reply:
최대한 한 줄로 표현할려고 하다 보니 좀 복잡해졌네요. 말씀하신대로 수정하였습니다.

Reply:
혹시 에디터 어떤거 쓰시나요? 아마 대부분의 에디터에 자동으로 코드 보기 좋게 간격 조절해주는 기능이 있을꺼예요.

Reply:
InteliJ를 프로젝트 진행할 때 가끔 사용합니다. 지금 한 거는 기본 파이썬 프로그램을 사용하였습니다. 한 번 알아봐야겠네요.

Reply:
pycharm에서 black 사용하시면 편해요 ! 
https://black.readthedocs.io/en/stable/editor_integration.html
필요하시면 한 번 보시면 좋을 것 같아요 :)